src/main/java/rest/CategoryResource.java

```diff
@@ -0,0 +1,38 @@
+package rest;
+
+import dto.VehicleDto;
+import entities.Category;
+import exceptions.CategoryException;
+import exceptions.VehicleException;
+import services.CategoryService;
+
+import javax.inject.Inject;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Path("/category")
+public class CategoryResource {
+
+	@Inject
+	CategoryService categoryService;
+
+	@POST
+	@Path("/create")
```

Comment:
Waarschijnlijk handiger om ipv POST /create een POST te hebben op / (zonder create).
Is een GET om alle categorieën op te halen ook nodig?